kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -332,7 +343,11 @@
             message = this.$formatRelative(download.requested_at, { now: this.now });
             break;
           case 'FAILED':
-            message = this.coreString('downloadFailedWillRetry');
+            if (this.sourceDeviceIsAvailable(download)) {
```

Comment:
No need to change this, as it was like this when you got here - but could also use the early return pattern here in this switch statement.